WAW.API/Chat/Injection/ChatInjections.cs

```diff
@@ -0,0 +1,16 @@
+using WAW.API.Chat.Domain.Models;
+using WAW.API.Chat.Domain.Repositories;
+using WAW.API.Chat.Persistence.Repositories;
+using WAW.API.Chat.Domain.Services;
+using WAW.API.Chat.Services;
+
+namespace WAW.API.Chat.Injection;
+
+public class ChatInjections {
```

Comment:
You forgot to make this a `static class`.

```suggestion
public static class ChatInjections {
```